queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/rules_and_memorandum.py

```diff
@@ -27,22 +27,24 @@
 def update_rules(
     business: Business,
     filing: Filing,
-    rules_file_key: String
+    file_key: String,
+    file_name: String
 ) -> Optional[List]:
-    """Updtes rules if any.
+    """Updates rules if any.
 
     Assumption: rules file key and name have already been validated
     """
-    if not business or not rules_file_key:
+    if not business or not file_key:
         # if nothing is passed in, we don't care and it's not an error
         return None
 
-    rules_file = MinioService.get_file(rules_file_key)
-    replace_file_with_certified_copy(rules_file.data, business, rules_file_key, business.founding_date)
+    rules_file = MinioService.get_file(file_key)
+    replace_file_with_certified_copy(rules_file.data, business, file_key, business.founding_date)
 
     document = Document()
     document.type = DocumentType.COOP_RULES.value
-    document.file_key = rules_file_key
+    document.file_key = file_key
+    document.file_name = file_name
```

Comment:
We need the file name to be stored, that way when we call the business endpoints, we can get the display name back.

`document.created` should be automatically populated I'm hoping.